tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetCore.cs

```diff
@@ -76,9 +84,8 @@ private static Assembly ResolveAssembly(string name)
             }
 
             // WARNING: Logs must not be added _before_ we check for the above bail-out conditions
-            StartupLogger.Debug("Assembly Resolve event received for: {0}", name);
             var path = Path.Combine(ManagedProfilerDirectory, $"{assemblyName.Name}.dll");
-            StartupLogger.Debug("Looking for: {0}", path);
+            StartupLogger.Debug("Assembly Resolve event received for: {0}. Looking for: {1}", name, path);
```

Comment:
Path.Combine can throw. It shouldn't but it can, it would be better to write the log before the Path, just to get more info in that case.